docs/settings.rst

```diff
@@ -45,6 +45,7 @@ DJANGO_AWS_ACCESS_KEY_ID                AWS_ACCESS_KEY_ID           n/a
 DJANGO_AWS_SECRET_ACCESS_KEY            AWS_SECRET_ACCESS_KEY       n/a                                            raises error
 DJANGO_AWS_STORAGE_BUCKET_NAME          AWS_STORAGE_BUCKET_NAME     n/a                                            raises error
 DJANGO_AWS_S3_REGION_NAME               AWS_S3_REGION_NAME          n/a                                            None
+DJANGO_GCE_STORAGE_BUCKET_NAME          GS_BUCKET_NAME              n/a                                            raises error
```

Comment:
I think we should also document the environment variable `GOOGLE_APPLICATION_CREDENTIALS` with Django setting n/a.

https://github.com/pydanny/cookiecutter-django/pull/2012/files#diff-78868204609793ad60885e85e4a879e6

```suggestion
DJANGO_GCE_STORAGE_BUCKET_NAME          GS_BUCKET_NAME              n/a                                            raises error
GOOGLE_APPLICATION_CREDENTIALS          n/a                         n/a                                            raises error
```

Reply:
Good call